src/schema/nodes/bibliography_item.ts

```diff
@@ -14,16 +14,32 @@
  * limitations under the License.
  */
 
-import { BibliographicDate, BibliographicName } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
 import { ManuscriptNode } from '../types'
 
-interface Attrs {
+type LooseNumber = string | number
+
+export interface BibliographyItemAuthor {
+  given?: string
+  family?: string
+  literal?: string
+  _id: string
+  objectType: 'MPBibliographicName'
+}
+export interface BibliographyItemDate {
+  'date-parts'?: [
+    [LooseNumber, LooseNumber?, LooseNumber?],
+    [LooseNumber, LooseNumber?, LooseNumber?]?
+  ]
+  _id: string
+  objectType: 'MPBibliographicDate'
+}
+export interface BibliographyItemAttributes {
```

Comment:
Let's call this `BibliographyItemAttrs` for consistency with similar things done in body-editor.

Reply:
Sure, done.